src/main/java/lesson9Refactoring1/Employee.java

```diff
@@ -0,0 +1,60 @@
+package lesson9Refactoring1;
+
+import java.math.BigDecimal;
+
+public class Employee {
```

Comment:
W tej klasie istnieje ogromne zagrożenie otrzymania NullPointerException. Zwróć szczególną uwagę na getTotalSalary

Reply:
czy w takim razie ustawić baseSalary na 0 domyślnie?

Reply:
czy po poprawkach jest juz ok? czy trzeba zastosowac jakaies 'try+catch'?

Reply:
Nie potrzeba tutaj 'try+catch'. Myślę, że domyślne ustawienie wartości na 0 to dobry pomysł. 
Zamiast `BigDecimal.valueOf(0)`można zastosować stałą z klasy BigDecimal: `BigDecimal.ZERO`

Reply:
dobra uwaga, dzieki!